keras/layers/__init__.py

```diff
@@ -1,24 +1,128 @@
 from __future__ import absolute_import
 
-from ..utils.generic_utils import deserialize_keras_object
```

Comment:
Wouldn't this be safe to have here?

Reply:
I'm not fond of having undocumented functions in public namespaces but ok.